### PR TITLE
Add a changelog, build improvements

### PR DESCRIPTION
**Pull request by wmdietl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,14 +59,14 @@ java \
     -dest /some/env/
 ```
 
-Download the eisop version of the Checker Framework and the default CF Language
+Download the typetools version of the Checker Framework and the default CF Language
```
Comment (Ao-senXiong) [inline README.md:62]: why we are using typetools again?

Comment (Ao-senXiong) [reply README.md:62]: thanks for the explainlation

**Review by Ao-senXiong — COMMENTED** [review]
Thanks, I am good with this PR now.

**Review by Ao-senXiong — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Version 0.3.0 (August 29, 2023)
```
Comment (wmdietl) [inline CHANGELOG.md:1]:
```suggestion
Version 0.3.0 (September 1, 2023)
```

**Pull request merged**